utilities/get_marker_key_list_for_joint.py

```diff
@@ -0,0 +1,13 @@
+import string
+
+
+def get_marker_key_list_for_joint(qualisys_dict: dict,
```

Comment:
pretty sure this function isn't necessary - dictionaries can return their keys directly as an interable

Reply:
see code sample above